packages/react/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # @openfun/cunningham-react
 
+## 0.10.0
+
+### Minor Changes
+
+- 5c61413: Add calendar system customization in `DatePicker` component
+- e4c1df5: fix datagrid column unique key
+- 0d6b98e: Introduce a DateRangePicker component
+- 0775490: Introduce a common DatePickerAux component
+- 114d0b5: Refactor DatePicker component
```

Comment:
nit: should we open a discussion about changeset linting? I might have missed some team's conventions, please feel free to share any useful resources.

Reply:
Yes I think you're right!